src/configurator/lib/knobs.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+ /**
+  * This kinda turned into something like storybook.js and similar. Maybe we
+  * should just use another library.
```

Comment:
What about Peter's story library?

Reply:
Yeah, I mentioned that in the PR description. If we opened sourced that, I would prefer to use it.